crates/sui-swarm/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
```

Comment:
Man I take the time to actually document a good portion of the api and you still want more! :D

Reply:
Yeah naming is hard. I built a similar thing for diem and there we had "process", so nodes running in isolated processes, and "k8s" nodes running on a k8s cluster.

Reply:
> I'd love for this technique to be adapted to Narwhal, as we will need it for e.g. integration testing. A Readme would help developers catch on to the main components they need to put in place (isolation, cancellation, configuration & launch)

Yeah i expect a lot of the building blocks could be applied to narwhal as well. It would even be awesome if we could uses common set of primitives, although i'm going to leave that for another time :)